examples/pom.xml

```diff
@@ -7,13 +7,13 @@
   <parent>
     <groupId>io.dapr</groupId>
     <artifactId>dapr-sdk-parent</artifactId>
-    <version>0.5.0-SNAPSHOT</version>
+    <version>0.6.0-SNAPSHOT</version>
```

Comment:
0.7.0?

Reply:
No, the Java-SDK is in a separate version.

Reply:
0.7 is the next version, we are releasing 0.5. This change is to update master.